packages/driver/src/cy/commands/agents.ts

```diff
@@ -167,7 +167,7 @@ export default function (Commands, Cypress, cy, state) {
     const log = Cypress.log({
       instrument: 'agent',
       name,
-      type: name,
+      type: name, // pretty sure this results in an invalid type.
```

Comment:
Should we do something about it?

Reply:
I dug into this a big. This is being leveraged by the Agent UI, not the Command UI which was where the initial confusion came in since there are 3 instrument types and 4 UI classes leverage it as the base-class with a slimmer usage of the full available options. Updated the Agent UI to reference the name property on the class since that better aligns with the other usage in the code base.

https://github.com/cypress-io/cypress/pull/20202/commits/ae43e364cce714a4f4edd2d949a10676a5453461